internal/commands/config/list_profiles_internal.go

```diff
@@ -1,21 +1,44 @@
 package config_internal
 
 import (
+	"slices"
+
+	"github.com/fatih/color"
+	"github.com/pingidentity/pingcli/internal/logger"
 	"github.com/pingidentity/pingcli/internal/output"
 	"github.com/pingidentity/pingcli/internal/profiles"
 )
 
 func RunInternalConfigListProfiles() {
+	l := logger.Get()
+
 	profileNames := profiles.GetMainConfig().ProfileNames()
 	activeProfile := profiles.GetMainConfig().ActiveProfile().Name()
 
 	listStr := "Profiles:\n"
+
+	slices.Sort(profileNames)
+
+	output.SetColorize()
+
+	activeFmt := color.New(color.Bold, color.FgGreen).SprintFunc()
+
 	for _, profileName := range profileNames {
 		if profileName == activeProfile {
-			listStr += "- " + profileName + " (active)\n"
+			listStr += "- " + profileName + activeFmt(" (active)") + " \n"
 		} else {
 			listStr += "- " + profileName + "\n"
 		}
+
+		description, err := profiles.GetMainConfig().ProfileViperValue(profileName, "description")
+		if err != nil {
+			l.Warn().Msgf("Cannot retrieve profile description for profile %s: %s.", profileName, err)
```

Comment:
Do we want this output to stdout as well? if so we can use a output.Print() with a result = ENUM_RESULT_NOACTION_WARN